Neo4j.Driver/Neo4j.Driver/Internal/ConnectionPool.cs

```diff
@@ -378,12 +378,22 @@ public Task CloseAsync()
 
             return Task.CompletedTask;
         }
-        
-        public async Task VerifyConnectivityAsync()
+
+        public async Task<IServerInfo> VerifyConnectivityAndGetInfoAsync()
         {
-            // Establish a connection with the server and immediately close it.
-            var connection = await AcquireAsync(Simple.Mode, Simple.Database, null, Simple.Bookmark).ConfigureAwait(false);
-            await connection.CloseAsync().ConfigureAwait(false);
+            var connection = await AcquireAsync(AccessMode.Read, null, null, null)
+                .ConfigureAwait(false) as IPooledConnection;
```

Comment:
Is there a benefit to using the _as_ keyword with _var_ rather than typing the variable? e.g. _IPooledConnection connection =_ ...

I genuinely don't know. Other than stylistic choice.

Reply:
I also prefer to use var for all local types, so you don't get a stylistic mix of type names and var.